### ISSUE TITLE
[Metricbeat] `consumer_utilisation` issue with RabbitMQ module in 7.0.0

### ISSUE THREAD

**Issue by sygnowskip** [elastic/beats#11833]
Hi,
after update of Metricbeat from version 6.6.2 to 7.0.0 module for RabbitMQ monitoring is not working. 

I'm receiving following errors (multiple):
`1 error: wrong format in "consumer_utilisation": expected integer, found <nil>;`

Configuration for `rabbitmq.yml`:
```
- module: rabbitmq
  metricsets: ["queue"]
  enabled: true
  period: 180s
  hosts: ["localhost:15672"]
  fields.beattype: "rabbit"
```

I've verified it and RabbitMQ is returing `consumer_utilisation` as `null`, but version 6.6.2 is able to parse it correctly, but 7.0.0 is throwing an error.

Is it expected behavior? Should I change something in configuration?

**Comment by jcoeltjen** [comment]
Same issue here.
The used schema for the object received by from the RabbitMQ management API did not change.
https://github.com/elastic/beats/blob/v6.6.2/metricbeat/module/rabbitmq/queue/data.go

Maybe the implementation of [schema](https://github.com/elastic/beats/blob/master/libbeat/common/schema/schema.go) 
or one of the helpers in [mapstriface](https://github.com/elastic/beats/blob/master/libbeat/common/schema/mapstriface/mapstriface.go) did change.

**Comment by odacremolbap** [comment]
@sygnowskip  @jcoeltjen 

I'm trying to reproduce the issue, no luck so far.
I guess the reason is that I'm not receiving the field, vs receiving it with no data.
Is that your case?

**Comment by odacremolbap** [comment]
able to reproduce, dismiss my previous comment

**Comment by odacremolbap** [comment]
@sygnowskip @jcoeltjen fix for this issue has been merged.
We expect it to be released 7.2
